invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/recordManagement.js

```diff
@@ -30,14 +29,22 @@ export const RecordManagement = (props) => {
           <span>Manage</span>
         </Grid.Row>
         <Grid.Row className="record-management-row">
-          { permissions.can_edit && <EditButton recid={recid} onError={handleError} /> }
-          { permissions.can_edit && <NewVersionButton recid={recid} onError={handleError} /> }
-        </Grid.Row>
-        { error &&
-        <Grid.Row className="record-management-row">
-          <p>{error}</p>
+          {permissions.can_edit && (
+            <EditButton recid={recid} onError={handleError} />
+          )}
+          {permissions.can_new_version && (
```

Comment:
Shouldn't this one be `can_edit` so that it is displayed?

Reply:
There is a specific permission for the `new_version` action :)

Reply:
... I am wondering how we can show the button disabled if the condition for showing the button in the first place is the same as the one for disabling it (i.e. wouldn't the button not show up at all?). Does that make sense?

Reply:
You are right, my bad :( I didn't refactor this. I will create the PR now :)

Reply:
@fenekku https://github.com/inveniosoftware/invenio-app-rdm/pull/710 . Maybe we should make the `EditButton` behave the same...